src/javascript/binary/websocket_pages/user/ico_subscribe.js

```diff
@@ -102,7 +109,7 @@ const ICOSubscribe = (() => {
                     { selector: '#price',    validations: ['req', ['number', { type: 'float', decimals: `1, ${decimal_places}`, min: Math.pow(10, -decimal_places).toFixed(decimal_places) }]], no_scroll: 1 },
 
                     { request_field: 'buy', value: 1 },
-                    { request_field: 'amount',        parent_node: 'parameters', value: () => document.getElementById('price').value },
+                    { request_field: 'amount',        parent_node: 'parameters', value: () => `${document.getElementById('price').value}/${initial_deposit_percent}` },
```

Comment:
same here, is it worth doing `+document.getElementById('price').value` and `+initial_deposit_percent`?

Reply:
@negarn This is supposed to be a string of format `1.35/5`.

Reply:
oops :)